docs/site/development/development-process.md

```diff
@@ -0,0 +1,292 @@
+    
+3.  **assess:** ~~Tickets marked "assess" need to be evaluated for priority (or whether to do them at all). Send your suggested disposition to the cldr group by email.~~
+    
+    1.  **duplicate:** If the ticket is a duplicate, any extra information is copied to the other ticket, an xref is set in both, and the ticket is returned.
+        
+    2.  **splitting:**
+        
+        1.  If the ticket requires separate phases, such as a change to the spec, then a separate ticket is filed, with xrefs set in both.
+                
+
+        2.  The owner can carry over all the triage acceptance including Owner, Component, etc.
+            
+            1.  If part of a ticket cannot be completed in the release:
+                
+                1.  Set the milestone to the release number
+                    
+                2.  Add a comment indicating what part of the ticket was done, and what part wasn't.
+                    
+                3.  Set a reviewer.
+                    
+                4.  Create a new ticket for the remaining work, with an xref to the old bug.
+                    
+
+    3.  **problems:**
+        
+
+        1.  If the ticket can't be done due to lack of time, or is being downgraded in priority from major or above to below major, send an email to the cldr group.
+                
+        2.  If during the course of doing the ticket, the person runs up against a problem, then send a message to the cldr group. Examples:
+                
+
+            *   The description doesn't work
+                
+            *   Another mechanism looks better
+                
+
+### DTD Changes
+
+*   **If you are making any DTD changes, please follow the instructions on** [**Updating DTDs**](/development/updating-dtds)**.**
+    
+
+### Testing!
+
+*   Add unit tests for changes you make, then
+    
+*   **Make sure to run the tests in** [**Running Tests**](/development/running-tests)**!**
+    
+
+Skipping test failures
+
+If there is a test failure that is due to a bug that cannot be fixed right now (eg it requires a data change, but it is past the data freeze), then use the logKnownIssue() method to disable the test. See [http://site.icu-project.org/setup/eclipse/time](http://www.google.com/url?q=http%3A%2F%2Fsite.icu-project.org%2Fsetup%2Feclipse%2Ftime&sa=D&sntz=1&usg=AOvVaw1SrxsiozTNKEAoaq7owsCr) for details, and look in the CLDR test code for some examples of usage.
+
+### Survey Tool in Production Phase!
+
+1.  **Before** you make a commit, add a line to the Issues spreadsheet below ([Spreadsheet View](https://docs.google.com/spreadsheet/ccc?key=0AqRLrRqNEKv-dF9DdVRPVmhHdzhheDliOEpLSUhObXc#gid=3)), with status OPEN, bug number, short desc.
```

Comment:
@macchiati Higher level comment: this is actually an obsolete process.

Reply:
Right. We just want to get the source into markdown before fixing.